build/arm/linker/nrf52840/softdevice.ld

```diff
@@ -20,10 +20,10 @@ SOFTDEVICE_MINIMUM_RAM_BASE = 0x1628;
 SOFTDEVICE_MINIMUM_CODE_BASE = 0x26000;
 
 /*
- * Reserving 16K in total. The more concurrent BLE links, the more RAM is required for SoftDevice.
+ * Reserving 23K in total. The more concurrent BLE links, the more RAM is required for SoftDevice.
  * Referenced by link.ld for system-part1
  */
-APP_RAM_BASE = 16K;
+APP_RAM_BASE = 23K;
```

Comment:
Just to confirm. Is this the minimum we need to support 3 central connections?

Reply:
Yes, by referencing the end address of the SoftDevice RAM that is printed over the logging interface.